src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="apple A1419 imac-27 Retina 5K monoblock"
+        class="product-card__image"
+      >
+      <h2 class="product-card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <h3 class="product-card__product-code">Product code: 195434</h3>
+
+      <section class="reviews">
```

Comment:
Don't create separate BEM block for reviews
It will always the element of the card